app/src/mainwindow2.cpp

```diff
@@ -699,8 +702,22 @@ void MainWindow2::importImageSequence()
         return;
     }
 
+    // Flag this so we don't prompt the user about auto-save in the middle of the import.
+    mIsImportingImageSequence = true;
+
     QStringList files = imageSeqDialog->getFilePaths();
     int number = imageSeqDialog->getSpace();
+
+    // Show a progress dialog, as this can take a while if you have lots of images.
+    QProgressDialog progress(tr("Importing image sequence..."), tr("Abort"), 0, 100, this);
```

Comment:
You can use the `totalImagesToImport` as the maximum value of progress bar instead of 100, so the Qt progress bar will do the % calculation for you. And updating the progress by `progress.setValue((imagesImportedSoFar);`, no need to do `progress.setValue((imagesImportedSoFar + 1)*progressMax / totalImagesToImport);`

Not a big issue, just tell a simpler way.